CooperationHullMainSite/Views/Home/HelpToAttend.cshtml

```diff
@@ -2,9 +2,13 @@
     ViewData["Title"] = "Help To Attend";
 }
 <div class="row">
-    <div class="col-12 content-text">
-        <h1 class="heading-xl">@ViewData["Title"]</h1>
+    <div class="col-12 ">
 
-        <p>Coming Soon</p>
+        <link href='https://actionnetwork.org/css/style-embed-whitelabel-v3.css' rel='stylesheet' type='text/css' />
```

Comment:
just been having a play around with this and I think it will be quite difficult to style it as we like _without_ any hacky CSS, since so many of the styles are provided by action network's stylesheet and overriding third party styles is notoriously tricky! When we don't include that stylesheet, it looks like the script is still imperatively updating style attributes on the DOM element upon interactions.

Is there any way we can write our own form HTML/CSS, and just POST to their API? It would be a little more work to set up, but I'd imagine we'd save time in the end as styling would be a lot simpler.

Reply:
We absolutely can, this one was thrown in as embedded to speed up phase 1 as @arthurHarding said it's not used much so it looking perfect wasn't a top priority.
If there isn't any better way to handle it then it should be ok as it is for now until I get the time to do it properly.

Reply:
yep, happy that this looks close enough for now. It won't get very much traffic